### DISCUSSION TITLE
Thank you!! (Also, Windows binaries...)

### DISCUSSION THREAD

**Discussion by Ander3232** [martinet101/ElevenClock#1150] [General]
Wow—a system-tray clock that actually WORKS with Windows 11!! 🌟😁🌟

Okay, I'm a bit overexcited—but I looked and looked, and never thought I'd find a Win-11-compatible replacement for Microsoft's inexplicably, persistently wimpy clock. 

Then I happened to see a reference to **ElevenClock** in a message forum. I t hought, "Okay, even though it has 'Eleven' it its name, don't get your hopes up.... It's probably yet another clock that won't work, or will look goofy, etc." I'm glad to be wrong!

BTW, after rambling around in your pages here—and being generally baffled, as usual, by GitHub's coder-oriented complexity)—I found ElevenClock Windows binaries here:

https://www.softpedia.com/get/Desktop-Enhancements/Clocks-Time-Management/ElevenClock.shtml

If you don't want to host them yourselves (not geeky enough, right?), you might take pity on us non-gurus and mention that they can be found there. 

Thanks again!

**Comment by marticliment** [comment]
Hi Ander,
ElevenClock is published on its official homepage: https://marticliment.com/elevenclock (though it does not appear on the top of Google yet, since I still need to work on my SEO.)

**Comment by Ander3232** [comment]
Hi Marti,
Thanks for your reply. (I was raving to you about how good ElevenClock was.)
Marti> ElevenClock is published on its official homepage: https://marticliment.com/elevenclock ...

Great! BTW, I learned about EC at Softpedia, and they have the wrong "Homepage" link for EC on their program page:
https://www.softpedia.com/get/Desktop-Enhancements/Clocks-Time-Management/ElevenClock.shtml
So you may want to let them know, too.
Marti > ...(though it does not appear on the top of Google yet, since I still need to work on my SEO.)

That must be tricky—I can't even imagine. 😯 Good luck with it, though—your work deserves a lot of attention, IMHO!

Cheers, A.

 - -
Andy Fielding
Technical Writer, EditorVancouver, BC, Canada
 

    On Saturday, March 25, 2023 at 02:39:38 a.m. PDT, Martí Climent ***@***.***> wrote:  
 
 


Hi Ander,
ElevenClock is published on its official homepage: https://marticliment.com/elevenclock (though it does not appear on the top of Google yet, since I still need to work on my SEO.)

**Comment by Ander3232** [comment]
Oh, sorry, Marti, I just wondered: Why does ElevenClock need so much space (~100 MB)? Does it install a large runtime package of some kind?

**Reply by marticliment** [reply]
It is basically 30mb python binaries and 70mb Qt library

**Reply by Ander3232** [reply]
That's a lot of code! Not as much as some software these days, though... In Linux, they use a system called Flatpak, where each app includes all its dependencies (runtimes, libraries, etc.)—so even a small app can end up installing over 2 GB of data (which you may already have, elsewhere, on your system. 😯 But storage is so big and cheap now, I guess that's the way people are going, so developers have more control over dependencies.

**Reply by marticliment** [reply]
It is not really that, but rather Windows doesn't have a proper way to manage shared dependencies such as apt or flatpak.
Also, DLL sharing does is not really ideal on Windows systems (crashes, errors, etc.)

**Comment by Ander3232** [comment]
Never mind, Marti, I sent Softpedia your new homepage URL. I'm always updating them on one thing or another... It makes me feel useful. 😁  

– A.

**Reply by marticliment** [reply]
Thanks!

**Comment by Ander3232** [comment]
Hi again Marti,
It's me—you know, the guy who bugs you all the time...? 😄
As you may have noticed from my sig, I'm a technical writer here in Canada. I'm so grateful for ElevenClock, I thought I'd send you some ideas for your EC webpage.
On-screen text is much harder to read than printed text—so usually, "less is more". If you keep your text short and clear, it's more likely that people will make the effort to read it.

Here are two PDFs: one that shows my changes, and one that shows just the new text.
Of course you're welcome to use, or not use, any of my ideas you wish. If you have questions about anything, feel free to ask.

Hope this helps! EC is a great project, and I think it deserves to look as good as possible.
Your fan, Andy
P.S.: I like your Python multi-program too.  😉

 - -
Andy Fielding
Technical Writer, EditorVancouver, BC, Canada

**Reply by marticliment** [reply]
Hello @Ander3232, 

I really appreciate your help, and it really helps me, since sometimes it is hard to find out the correct words. I am willing to introduce the changes to ElevenClock website
However, i can't see the PDF files you sent... could you please send them to marticlilop@gmail.com? GitHub does not handle very well attached files.

Thank you!

Martí Climent

**Reply by Ander3232** [reply]
Sorry, man, I just saw your messages. I'll send that to you again, right away. Glad I could help!

**Reply by marticliment** [reply]
I have received them. Thanks!
